### ISSUE TITLE
Time Tracking?

### ISSUE THREAD

**Issue by Curotos** [mpm11011/galaxy-integration-rpcs3#17]
Is there a way to have gog track the rpcs3 game time?

**Comment by massimilianodelliubaldini** [comment]
Of course, it should already be part of the plugin. Note that it only works when you launch a game from Galaxy, not from rpcs3 directly. Are you saying that's not working?

**Comment by Curotos** [comment]
Yeah whenever I launch an rpcs3 game from gog it does not track the game time, I added another rpcs3 game as a bat file because it didn't show up with the plugin and its tracking time for that one but not any of the games pulled from the plugin?

**Comment by massimilianodelliubaldini** [comment]
Ok, edit this issue and use the issue template provided. You'll need to upload the logs requested there.

**Comment by Curotos** [comment]
Wait i'm sorry what should I do?

**Comment by massimilianodelliubaldini** [comment]
On [this page](https://github.com/mpm11011/galaxy-integration-rpcs3/issues/new/choose) click Get Started, copy all the text in there, reply to this issue, and fill in all the requested information.

**Comment by perkel666** [comment]
I can also confirm it doesn't track time. GOG seems to not see game that is run and after play it imminently shows up violet instead of grey

Using latest rpcs3 on latest win10 with latest integration version using colecovision from integration menu as there is no RPCS3 integration

**Comment by NoFear99354** [comment]
Isn't working for me either.

**Comment by Dayflare** [comment]
i fixed time tracking with this PR: https://github.com/mpm11011/galaxy-integration-rpcs3/pull/28
